docs/reference/module-options.md

```diff
@@ -0,0 +1,148 @@
+# Module configuration options
+
+Most Apostrophe modules, both core and project-level, can be configured with settings that influence functionality without having to write custom Javascript. The sections below describe the options available in all modules as well as those specific to certain module types.
+
+All the settings described here are placed in a module's `options` configuration object. This often added in the respective `index.js` file, but can also be added where the module is instantiated in the `app.js` file.
```

Comment:
"The `options` object can be added in the module's `index.js` file, as well as where the module is instantiated in the `app.js` file.

Remove "the" in "All the settings"